packages/next/README.md

```diff
@@ -1108,6 +1108,7 @@ export default class MyDocument extends Document {
   }
 
   render() {
+    // Important: All of Head, Main, and NextScript are required to be rendered
```

Comment:
I still agree with @rafaelalmeidatk, regarding that the info about the required components being outside the code example.